recipes/openvdb/all/conanfile.py

```diff
@@ -130,10 +130,10 @@ def package_id(self):
 
     def requirements(self):
         # https://github.com/AcademySoftwareFoundation/openvdb/blob/v10.0.1/doc/dependencies.txt#L36-L84
-        self.requires("boost/1.84.0", transitive_headers=True)
+        self.requires("boost/1.85.0", transitive_headers=True)
         self.requires("onetbb/2021.10.0", transitive_headers=True, transitive_libs=True)
         if self.options.use_imath_half:
-            self.requires("imath/3.1.9", transitive_headers=True, transitive_libs=True)
+            self.requires("imath/3.1.11", transitive_headers=True, transitive_libs=True)
```

Comment:
Most recipes on CCI still rely on `3.1.9` so I would suggest sticking to that one here and in the OpenUSD recipe to avoid conflicts, unless you have a specific reason to use the newer version. See https://github.com/conan-io/conan-center-index/issues/23277

Reply:
Thanks for your feedback.